build-logic/src/main/kotlin/me.champeau.convention-testing.gradle.kts

```diff
@@ -7,12 +7,8 @@ val isCiBuild = providers.environmentVariable("CI").isPresent
 tasks.withType<Test>().configureEach {
     useJUnitPlatform()
 
-    maxParallelForks = if (isCiBuild) {
-        Runtime.getRuntime().availableProcessors()
-    } else {
-        // https://docs.gradle.org/8.0/userguide/performance.html#execute_tests_in_parallel
-        (Runtime.getRuntime().availableProcessors() / 2).takeIf { it > 0 } ?: 1
-    }
+    // https://docs.gradle.org/8.8/userguide/performance.html#execute_tests_in_parallel
+    maxParallelForks = (Runtime.getRuntime().availableProcessors() / 2).coerceAtLeast(1)
```

Comment:
Why reducing the limit on CI?

Reply:
GHA upgrades to 4-core runners (2-core before), now we can just use half of it instead of the full, to keep the guide from https://docs.gradle.org/8.8/userguide/performance.html#execute_tests_in_parallel

Reply:
Sure but why limiting?

Reply:
I'm unsure if there is any risk in keeping the previous config. Anyway, I can revert this if you want.

Reply:
I don't think there would be, the goal was to reduce pressure on local machines, but for CI we don't care.

Reply:
Dropped.